.github/workflows/build.yml

```diff
@@ -1,27 +1,44 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
-
-name: Python package
+name: Build
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
 
 jobs:
   build:
 
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, pypy3]
```

Comment:
What does this change mean in practice (some versions are dropped)?

Reply:
Versions in this file are indicated along job definitions under `include`.

But these versions have actually been moved to the new `tests.yml` file.